src/styles/blocks/variables.scss

```
@@ -0,0 +1,7 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+$text-color:#616070;
```

it will be better to separate imports and variables by empty line and add some space after semicolon `:` for variables - it will be more clear
```suggestion

$text-color: #616070;
```